views/addTag.ejs

```diff
@@ -0,0 +1,7 @@
+<h1>Add tag for <%= favorites.title %></h1>
+<form method="POST">
+	<div class="form-group">
+		<input type="text" name="tag" class="form-control" placeholder="Your tag" />
+	</div>
+	<input type="submit">
+</form>
```

Comment:
After this form, let's add code to show all currently listed tags for this favorite:
```
<% tags.forEach(function(tag){ %>
	<p>#<a href="/tag/<%= tag.id %>"><%= tag.tag %></a></p>
<% }); %>
```